README.md

```diff
@@ -1,28 +1,31 @@
-# Unreal GDK Third-Person Shooter (pre-alpha)
+# SpatialOS GDK Third-Person Shooter game
 
-Use of the contents of this repository is subject to the [Improbable license](LICENSE.md).
+* **LICENSE:** Use of the contents of this repository is subject to the [Improbable license](LICENSE.md)
+* **REPOSITORY:** github.com/spatialos/UnrealGDKStarterProject
 
-Note that this project and the contents of this repository are in a **pre-alpha stage**, meaning that they are meant for evaluation only.
 
-## Introduction
+This project is a sample third-person shooter deathmatch game which uses the [SpatialOS GDK for Unreal](github.com/spatialos/unrealGDK). 
 
-This project is a sample third-person shooter deathmatch game implemented on top of the SpatialOS Unreal GDK. It is meant to be a reference project and/or a starting point for those building games on SpatialOS and Unreal Engine 4. In the game, you can join a team, run around, shoot characters on other teams, and spawn "test cubes" (cubes you can shoot around).
+It is both a reference project and a starting point for those building games on SpatialOS and Unreal Engine 4. In the game, you can join a team, run around, shoot characters on other teams, and spawn "test cubes" (which are cubes you can shoot around).
 
-For setup instructions, follow [the guide in the SpatialOS Unreal GDK repository](https://github.com/improbable/UnrealGDK/blob/master/docs/setup-and-installing.md), replacing the "Starter Project" repository with this one.
 
-For more information, see the [SpatialOS Unreal GDK repository](https://github.com/improbable/UnrealGDK) and its [documentation](https://github.com/improbable/UnrealGDK/blob/master/docs/readme.md).
+For setup instructions, follow the [Get started](https://docs.improbable.io/unreal/latest/get-started/introduction) guide in the GDK for Unreal documentation. This guide uses the [GDK for Unreal Starter Project](https://github.com/spatialos/UnrealGDKStarterProject) as an example. When you follow the installation instructions, you need to replace the "Starter Project" repository with this one.
```

Comment:
Should this link to https://docs.improbable.io/unreal/pre-alpha/setup-and-installing? (Current link is dead)

Also, how come this is an absolute link rather than relative? Is it to do with getting the docs from GitHub onto the main docs site?